src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/PedigreeAnnotation.java

```diff
@@ -46,14 +48,18 @@ public PedigreeAnnotation(final GATKPath pedigreeFile){
         initializeSampleDBAndSetFounders(pedigreeFile);
     }
 
+    protected SampleDB getSampleDB() {
+        final SampleDBBuilder sampleDBBuilder = new SampleDBBuilder(pedigreeValidationType);
+        sampleDBBuilder.addSamplesFromPedigreeFiles(Collections.singletonList(pedigreeFile));
+
+        return sampleDBBuilder.getFinalSampleDB();
+    }
```

Comment:
If the constructor overload suggestion works, hopefully this can be removed.

Reply:
So there is actually some utility is exposing a SampleDB getting to subclasses (my code would use it); however, if there is a pedigreeFile getter there isnt a lot of difference between exposing a method to create the SampleDB or doing this in my code. If you feel strongly we can make that private.

Reply:
**By 'hope it can be removed', I assume you just mean exposing the getter. That code existed before. This method is just consolidating two pieces of redundant code.

Reply:
The File and validation getters are fine. Its the SampleDB stuff that we want to remove from this class (see the branch I referred to).

Reply:
OK, i see that now.

Then you might not like what I just proposed on constructors. If I add PedigreeValidationType to overload constructors, this becomes a bit more of a mess with all sorts of constructor versions. I proposed simplifying this to two constructors: one with nullable founderIds and nullable pedigree, and another with those two plus pedigreeValidationType. It seemed much clearer than 4 flavors. If you ultimately separate about founderIds then I would think that could be backed out without much complication?